tests/test-dirs/syntax-document/language-extensions.t/run.t

```diff
@@ -0,0 +1,82 @@
+// Recursive defition of functions
+  $ $MERLIN single syntax-document -position 1:6 \
+  > -filename ./recursive-def.ml < ./recursive-def.ml
```

Comment:
Where are the tests results ?  You should run the test and then `dune promote` the results :-) 

Also, it make the tests hard to read to have every case in a different file and all the commads in a single other.
Don't feel obliged to change it now, but it would have been clearer to use the `cat >***.ml <<'EOF'` syntax you can find in many other tests to have the content of the file displayed right before the call to Merlin. Like in this test for example:

https://github.com/ocaml/merlin/blob/master/tests/test-dirs/document/unattached-comment.t#L4-L16

Reply:
Yes, this was the initial structure I was using. If it's better I'll revert back to it.